frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -557,37 +532,14 @@ private APIGatewayProxyResponseEvent handleInvalidCredentials(
         }
 
         if (decision instanceof Decision.TemporarilyLockedOut) {
-            if (isReauthJourney) {
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.AUTH_REAUTH_FAILED,
-                        auditContext.withSubjectId(authSession.getInternalCommonSubjectId()),
-                        ReauthMetadataBuilder.builder(userPermissionContext.rpPairwiseId())
-                                .withAllIncorrectAttemptCounts(
-                                        getReauthAttemptCounts(
-                                                journeyType,
-                                                userPermissionContext.internalSubjectId(),
-                                                userPermissionContext.rpPairwiseId()))
-                                .withFailureReason(ReauthFailureReasons.INCORRECT_PASSWORD)
```

Comment:
I think the withFailureReason (now on line 520) should use the decision payload and anticorruption layer to get the right reason. Recognise this was there before the PR

```
ReauthFailureReasons reauthFailureReason =
                    ForbiddenReasonAntiCorruption.toReauthFailureReason(	                    ForbiddenReasonAntiCorruption.toReauthFailureReason(
                            reauthLockedOut.forbiddenReason());	                            reauthLockedOut.forbiddenReason());
                            ```